install/mysql/glpi-empty.sql

```diff
@@ -9923,4 +9891,16 @@ CREATE TABLE `glpi_assets_assettypes` (
   KEY `date_creation` (`date_creation`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC;
 
+DROP TABLE IF EXISTS `glpi_groups_assets`;
+CREATE TABLE `glpi_groups_assets` (
+  `id` int unsigned NOT NULL AUTO_INCREMENT,
+  `groups_id` int unsigned NOT NULL DEFAULT '0',
+  `itemtype` varchar(255) NOT NULL DEFAULT '',
+  `items_id` int unsigned NOT NULL DEFAULT '0',
+  `type` tinyint NOT NULL DEFAULT '0',
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `unicity` (`groups_id`,`itemtype`,`items_id`),
+  KEY `item` (`itemtype`, `items_id`)
```

Comment:
Won't our own checker consider it "useless" with that field being included in the unicity index now?

Reply:
No, what is useless is an index that contains the first fields (in the same order) as another index, e.g. `groups_id` in the current case, because it is the first field of the unicity index.